src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.spec.ts

```diff
@@ -902,5 +902,52 @@ describe('CdValidators', () => {
         testValidator('testName', true);
       }));
     });
+
+    describe('endpointValidator', () => {
+      it('should return null for a valid URL with port', () => {
+        const control = new UntypedFormControl('https://example.com:8080');
+        expect(CdValidators.endpointValidator(control)).toBeNull();
+      });
+
+      it('should return null for multiple valid URLs with ports', () => {
+        const control = new UntypedFormControl('https://example.com:8080,http://localhost:3000');
+        expect(CdValidators.endpointValidator(control)).toBeNull();
+      });
+
+      it('should return an error object for a URL without a port', () => {
+        const control = new UntypedFormControl('https://example.com');
```

Comment:
url without port is a valid url isn't it? for ssl the default port is 443 and if its not given its assumed on its own. same way we don't say google.com:443. Why are we saying otherwise? Is there anything wrong with an implicit port here?

Reply:
Updated and removed the requirement of port